Sources/Sentry/include/HybridPublic/PrivateSentrySDKOnly.h

```diff
@@ -103,6 +104,11 @@ typedef void (^SentryOnAppStartMeasurementAvailable)(
  */
 + (NSDictionary *)getExtraContext;
 
+/**
+ * Allow Hybrids SDKs to set the current Trace.
+ */
++ (void)setTrace:(SentryId *)traceId spanId:(SentrySpanId *)spanId;
```

Comment:
`l`: We might want to add a comment to the comments if this method is thread-safe